docs/best-practices.md

```diff
@@ -155,6 +158,31 @@ Example:
 `include::encryption-types.adoc[leveloffset=+1]`
 (the including file in this example is in the same directory as the included file)
 
+## Table of Contents
+
+Prefix: `toc`
+
+Refernce: [`Table of Contents (ToC)`](https://asciidoctor.org/docs/user-manual/#user-toc)
+
+A table of contents (ToC) is, if not otherwise defined, an index of section and subsection
+titles that can be automatically generated from the pages structure when converting a
+document with Asciidoctor.
+
+The easiest way adding a ToC is shown in the following example.
+
+```
+= The Page Header
+:toc:
+
+Page contents.
+```
+Please also see additional directives like:
+[`toc-title`](https://asciidoctor.org/docs/user-manual/#user-toc-title),
+[`toc-level`](https://asciidoctor.org/docs/user-manual/#user-toc-levels) or
+[`In-Document Placement`](https://asciidoctor.org/docs/user-manual/#manual-placement)
```

Comment:
Why is the text for this link wrapped in backticks?

Reply:
Because we did on top of the page for all the other references too and the page renders in github correct

Reply:
But the text is normal wording, not a technical term, nor reference.

Reply:
I have tried without square brackets respectively without the ticks, but then it does not render correct. Leaving it as committed.